### ISSUE TITLE
Adding a Script for setting the Github Actions

### ISSUE THREAD

**Issue by i-am-yuvi** [sahil-sagwekar2652/GitHub-Automation-scripts#15]
I think we could also add a script for setting the GitHub Actions if the user requires it. Normally the user has to go to the specified repo and enable the Actions. I will create a default YAML file which will basically create a basic Github Actions for the project.

I want to work on it asap, kindly assign this to me.

**Comment by sahil-sagwekar2652** [comment]
Hey there @i-am-yuvi , happy to have you here! 

Just to confirm, your idea is to create a script which will create and push a sort of default Github actions workflow?

Join our slack server for further discussions - https://join.slack.com/t/githubautomat-9t49360/shared_invite/zt-1v05p5kao-UBmelVfZd6EBjGuzd_6XYg

**Comment by sahil-sagwekar2652** [comment]
The GSSoC'23 program managers are yet to disclose the scoring system and how to track contributions.
Don't worry, I will assign this to you once we get the instructions.

**Comment by i-am-yuvi** [comment]
Yes @sahil-sagwekar2652, I was thinking while create a new repo a simple workflow to be created in Github Actions. What do you think?

**Comment by sahil-sagwekar2652** [comment]
any updates @i-am-yuvi ? Let me know or I'll have to unassign you.
